src/app.js

```diff
@@ -59,6 +58,20 @@ JoystickLib.getJoystick()
 					return { x: -1, y: 0 };
 				case 'right':
 					return { x: 1, y: 0 };
+				case 'click':
+					return { x: 0, y: 0 };
+			}
+		}
+
+		// Given a value, return a value that is within the
+		// bounds of the LED matrix
+		function clamp(value) {
```

Comment:
I was going to add this to the README as a `Further tasks`, so that people at the hackathon have an easy task to get to grips with the codebase, what do you think @alisondavis17 @shaunmulligan @josephroberts ?

Reply:
I added this because it's very easy to lose the LED off the edge of the screen with no way to get it back. How about we remove it and make `click` centre the LED?

Reply:
That sounds good.